qiskit_ibm_runtime/options/simulator_options.py

```diff
@@ -47,3 +51,17 @@ class SimulatorOptions:
     seed_simulator: Optional[int] = None
     coupling_map: Optional[List[List[int]]] = None
     basis_gates: Optional[List[str]] = None
+
+    def from_backend(self, backend: Union[BackendV1, BackendV2]):
```

Comment:
```suggestion
    def set_backend(self, backend: Union[BackendV1, BackendV2]):
```

`from_backend` seems to be a class method. `set_backend` might be better.